api/gatewayserver.proto

```diff
@@ -69,6 +70,12 @@ service GtwGs {
 message ScheduleDownlinkResponse {
   // The amount of time between the message has been scheduled and it will be transmitted by the gateway.
   google.protobuf.Duration delay = 1 [(gogoproto.stdduration) = true, (gogoproto.nullable) = false, (validate.rules).duration.required = true];
+  // Downlink path chosen by the Gateway Server.
+  DownlinkPath downlink_path = 2;
+  // Whether RX1 has been chosen for the downlink message.
+  bool rx1 = 3;
+  // Whether RX2 has been chosen for the downlink message.
+  bool rx2 = 4;
```

Comment:
Technically, the same downlink message can be scheduled on the same downlink path on both RX1 and RX2: the gateway transmitting the same downlink message twice.